packages/@aws-cdk/aws-codebuild/test/test.project.ts

```diff
@@ -1466,12 +1466,393 @@ export = {
 
         test.done();
       },
+
+      'can be provided as a SecretValue from a SecretsManager imported Secret by complete ARN'(test: Test) {
+        // GIVEN
+        const stack = new cdk.Stack();
+
+        // WHEN
+        const secret = secretsmanager.Secret.fromSecretCompleteArn(stack, 'Secret',
+          'arn:aws:secretsmanager:us-west-2:123456789012:secret:mysecret-123456');
+        new codebuild.PipelineProject(stack, 'Project', {
+          environmentVariables: {
+            'ENV_VAR1': {
+              type: codebuild.BuildEnvironmentVariableType.SECRETS_MANAGER,
+              value: secret.secretValueFromJson('json-key'),
```

Comment:
This test is pointless, in my mind - it doesn't offer any additional capability by passing `SecretValue` here (you might as well provide the `value` as `'arn:aws:secretsmanager:us-west-2:123456789012:secret:mysecret-123456:jsonkey'`, and the effect would be the same).